bedrock/mozorg/templates/mozorg/about.html

```diff
@@ -125,9 +119,9 @@ <h1 class="title-shadow-box">{{ _('Get to know Mozilla') }}</h1>
   <aside id="newsletter-subscribe">
     <div class="content container">
     {% if LANG.startswith('en-') %}
-      {{ email_newsletter_form(newsletters='mozilla-foundation', title=_('Love the Web?'), subtitle=_('Get the Mozilla newsletter and help us keep it open and free.'), button_class='light') }}
+      {{ email_newsletter_form(newsletters='mozilla-foundation', title=_('Sign up. Read up.<br> Make a difference.'), subtitle=_('Get the Mozilla newsletter and help us keep the Web free and open.'), button_class='red') }}
```

Comment:
cc @Peiying - a localizable string changed here, but it's only shown for `en-` locales.